tests/servers/test_liveserverthread.py

```diff
@@ -0,0 +1,28 @@
+from django.db import DEFAULT_DB_ALIAS, connections
+from django.test import LiveServerTestCase, TestCase
+
+
+class LiveServerThreadTest(TestCase):
+
+    def run_live_server_thread(self, connections_override=None):
+        thread = LiveServerTestCase._create_server_thread(connections_override)
+        thread.daemon = True
+        thread.start()
+        thread.is_ready.wait()
+        thread.terminate()
+
+    def test_closes_connections(self):
+        conn = connections[DEFAULT_DB_ALIAS]
```

Comment:
Also, `LiveServerTestCase.setUpClass()` assigns to `connections_override` from `django.db.connections`, so this code is just doing the same thing as `LiveServerTestCase`.